Makefile

```diff
@@ -16,32 +16,38 @@ TESTS := false
 # The endpoint uses HTTP connection to transmit encrypted data by default.
 # See "HTTPS Connection Support" in docs/endpoint.md for more information.
 ENFORCE_EP_HTTPS := false
-# The CFLAGS to pass during endpoint app build
-ENDPOINT_CFLAGS :=
+# The CFLAGS to pass into endpoint build process for mkapp.
+# The mkapp is a tool provides by Legato project.
```

Comment:
Use "Legato Application Framework" or "Legato AF" instead of "Legato project", which is the umbrella of several open source projects initiated by Sierra Wireless.